mathics/core/pattern.py

```diff
@@ -3,8 +3,9 @@
 # -*- coding: utf-8 -*-
 
 
+from mathics.core.atoms import Integer
 from mathics.core.element import ensure_context
-from mathics.core.expression import Expression
+from mathics.core.expression import Expression, SymbolDefault
 from mathics.core.symbols import Atom, Symbol, system_symbols
 from mathics.core.systemsymbols import SymbolSequence
```

Comment:
I realize this was code from before. If you are up to it: 

`SymbolAlternatives`, `SymbolBlank`, `SymbolCondition`, `SymbolPattern` can be imported and removed from the initialization list on line 21  and the remaining items sorted.

Reply:
#613 attempts to do this.